### PR TITLE
Update context in `Menu`

### PR DESCRIPTION
**Pull request by jjenzz**
----------

- [x] Change base branch to `main` once #477 is merged. 

----------

I'll add versions file after the above is merged ☝️

### CODE REVIEW

**Review by benoitgrelard — COMMENTED** [review]
Same, just the name change.

**Review by benoitgrelard — APPROVED** [review]

File: packages/react/menu/src/Menu.tsx
Comment (jjenzz) [inline packages/react/menu/src/Menu.tsx:206]: I'll wait for #496 to merge and then update this before merging.

**Review by benoitgrelard — APPROVED** [review]

**Pull request merged**
